### PR TITLE
CC production checklist

### LINKED ISSUE #5630: CC: "Move into production" checklist
[Based on Tim O.'s feedback during the Sept. Education Product Council]

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Closes #5630

### CODE REVIEW

File: v19.2/cockroachcloud-production-checklist.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+build_for: [cockroachcloud]
+---
+
+This page provides important recommendations for CockroachCloud production deployments.
+
+## Follow the SQL Best Practices
+
+To ensure optimal SQL performance for your CockroachCloud cluster, follow the best practices described in the [SQL Performance Best Practices](performance-best-practices-overview.html) guide.
+
+## Use a connection pool
```
Comment (tim-o) [inline v19.2/cockroachcloud-production-checklist.md]: Maybe "Use a small pool of persistent connections" instead?

Comment (tim-o) [reply v19.2/cockroachcloud-production-checklist.md]: That's my only nit. Otherwise this looks great. Feel free to merge once that's updated.

Comment (Amruta-Ranade) [reply v19.2/cockroachcloud-production-checklist.md]: Fixed.

**Comment by Amruta-Ranade** [comment]
TFTR!

**Comment by Amruta-Ranade** [comment]
@lhirata I am merging this PR for now because the CC UI team is blocked on it. Please review it when you get a chance and feel free to reopen it if you find issues.

**Pull request merged**